src/snapred/backend/dao/request/FocusSpectraRequest.py

```diff
@@ -0,0 +1,11 @@
+from matplotlib import use
+from pydantic import BaseModel
+
+
+class FocusSpectraRequest(BaseModel):
```

Comment:
is this a merge hiccup?  This file already exists: https://github.com/neutrons/SNAPRed/blob/next/src/snapred/backend/dao/request/FocusSpectraRequest.py